sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.institutions.controller.ts

```diff
@@ -193,9 +192,7 @@ export class ConfirmationOfEnrollmentInstitutionsController extends BaseControll
       })),
       applicationPIRStatus:
         disbursementSchedule.studentAssessment.application.pirStatus,
-      disbursementDate: getExtendedDateFormat(
-        disbursementSchedule.disbursementDate,
-      ),
+      disbursementDate: disbursementSchedule.disbursementDate,
```

Comment:
is this `getExtendedDateFormat` taken care of by the vue side?

Reply:
Yes, it is converted at Vue side.